checkov/terraform/checks/resource/alicloud/PasswordPolicyMaxLogin.py

```diff
@@ -28,6 +28,8 @@ def scan_resource_conf(self, conf):
 
         if conf.get('max_login_attempts'):
             length = force_int(conf.get('max_login_attempts')[0])
+            if not length:
```

Comment:
```suggestion
            if length is None:
```
because `0` is a valid value 🙂